app/forms/calculators/guaranteed_income_form.rb

```diff
@@ -0,0 +1,24 @@
+require 'active_model'
+
+module Calculators
+  class GuaranteedIncomeForm
+    include ActiveModel::Model
+
+    attr_reader :pot, :age
+
+    validates :pot, presence: true, numericality: { allow_blank: true, greater_than: 0 }
+    validates :age, presence: true, numericality: { allow_blank: true, greater_than: 54, less_than: 76 }
```

Comment:
`54` and `76` are magic numbers. We should extract them to constants that reveal the significance of those numbers.